nss/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #[macro_use]
-extern crate lazy_static;
+extern crate lazy_static; // used by libnss_*_hooks macros
```

Comment:
Do you feel like this comment is important? (Very honest question)
The compiler will already yell at you if you don't declare this (and that's why it's also close to where we include the hooks from `libnss`)